003-examples.md

```diff
@@ -1,36 +1,35 @@
-Examples: Mapping VO-DML ⇒ VOTable
-==================================
+# Examples: Mapping VO-DML to VOTable #
 
 This section shows examples of mappings as they may occur in realistic
 VOTables, using example Data Models. We will describe how the different
 VO-DML elements must be serialized and how annotations employing the
-&lt;VODML&gt; element can help one interpret the VOTable. In the later
+<VODML> element can help one interpret the VOTable. In the later

```

Comment:
I would add a back tick here. I don't think there are enough instances of such tags referred directly in the text to require a special filter (see come comments below about **`ObjectTypes`** et al.